pkg/apis/kops/validation/aws.go

```diff
@@ -160,6 +186,7 @@ func awsValidateMixedInstancesPolicy(path *field.Path, spec *kops.MixedInstances
 	// @step: check the instance types are valid
 	for i, x := range spec.Instances {
 		errs = append(errs, awsValidateInstanceType(path.Child("instances").Index(i), x, cloud)...)
+		errs = append(errs, awsValidateInstanceTypeWithImageArch(path.Child("instances").Index(i), x, ig.Spec.Image, cloud)...)
```

Comment:
@rifelpet here I am adding the check for the ImageArch for mixed instance policies. Will be refactoring the code to remove awsValidateInstanceType

Reply:
ah my mistake, this lgtm 👍🏻